.github/workflows/lint.yml

```diff
@@ -22,5 +22,7 @@ jobs:
           python-version: 3.9
       - name: Install Dependencies
         run: pip install -e '.[dev]'
-      - name: Run Pytest
+      - name: Run linting check
         run: make lint
+      - name: Check formatting
+        run: make format --check
```

Comment:
```suggestion
      - name: Check formatting
        run: make lint
```

Reply:
i left it as previous, because people may want to use `make format` to actually apply the changes.